disnake/app_commands.py

```diff
@@ -357,6 +357,9 @@ def __repr__(self) -> str:
         attrs = " ".join(f"{key}={getattr(self, key)!r}" for key in self.__repr_info__)
         return f"<{type(self).__name__} {attrs}>"
 
+    def __str__(self) -> str:
+        return f"<{type(self).__name__} name={self.name!r}>"
```

Comment:
if we're using ``__str__``, why are we making this like a repr? This should be just the command name, as it is *not* a repr.

Reply:
if we want something specific for debug syncing printing, we should do something different.

Reply:
You wouldn't be able to tell apart the different types of commands if it just returned the command name, and knowing the type of command seems like something you'd want for debugging.
We could just remove `__str__` here entirely, and do this in the sync debug logging part of the code?

Reply:
https://github.com/DisnakeDev/disnake/pull/478/commits/e8bf2c03ee0c5473e98c40edf1d7c2eabc378b85